kevm_pyk/src/kevm_pyk/__main__.py

```diff
@@ -92,7 +98,7 @@ def exec_solc_to_k(
         with open(exclude_tests, 'r') as el:
             _exclude_tests = el.read().strip().split('\n')
     contract = Contract(contract_name, contract_json, foundry=False)
-    contract_module, contract_claims_module = contract_to_k(contract, empty_config, foundry=False, exclude_tests=_exclude_tests)
+    contract_module, contract_claims_module = contract_to_k(contract, empty_config, foundry=False, exclude_tests=_exclude_tests, imports=('VERIFICATION'))
```

Comment:
```suggestion
    contract_module, contract_claims_module = contract_to_k(contract, empty_config, foundry=False, exclude_tests=_exclude_tests, imports=('VERIFICATION',))
```

The type checker didn't complain as `str` is an `Iterable[str]`.

Reply:
Fixed, I actually removed the hardcoding of that name altogether.